createClass.html

```diff
@@ -6,8 +6,14 @@
 <body>
 	<h1>CREATE CLASS</h1>
 	<form id="xcelform" method="post">
-		Class Name:
-		<input type="text" id="className" required>
+		Class ID: =stream+location+onsite/offshore+date+srNo

```

Comment:
This shows the formula for class ID in the browser. That is ok for now, just be aware.